blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -1001,6 +1008,50 @@ fn sync_tipset<DB: BlockStore + Sync + Send + 'static, C: Consensus>(
     })
 }
 
+async fn fetch_batch<DB: BlockStore + Send + Sync + 'static, C: Consensus>(
+    batch: &[Arc<Tipset>],
+    network: &SyncNetworkContext<DB>,
+    chainstore: &Arc<ChainStore<DB>>,
```

Comment:
It's essentially always a bad idea to have a reference to an `Arc`. Either use `Arc<T>` or `&T`.
```suggestion
    chainstore: &ChainStore<DB>,
```